ircd/supported.c

```diff
@@ -300,6 +301,21 @@ isupport_nicklen(const void *ptr)
 	return result;
 }
 
+static const char *
+isupport_client_tag_deny(const void *ptr)
+{
+	static char result[200];
+	static char exceptions[198];
```

Comment:
`exceptions[]` does not need to be `static`.

Reply:
Fixed